VirtoCommerce.Platform.Tests/Common/QueryableExtensionsTests.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using VirtoCommerce.Platform.Core.Common;
+using Xunit;
+
+namespace VirtoCommerce.Platform.Tests.Common
+{
+    [CLSCompliant(false)]
+    public class QueryableExtensionsTests
```

Comment:
Please add     
[Trait("Category", "CI")] 
to include these tests into CI process

Reply:
fix